### DISCUSSION TITLE
Show URL that has been blocked

### DISCUSSION THREAD

**Discussion by danrot** [SafeExamBrowser/seb-win-refactoring#335] [Q&A] [answered]
I am trying to configure the SafeExamBrowser for one of my exams. I would like to only allow a minimum set of URLs, so that students are not able to open any other page than I intend to.

I've added the first rules, but when I open the SafeExamBrowser it shows the following message:

```
Access to "" is not allowed according to the current configuration.
```

Mind that the empty quotes are not because I want to hide some information, the error message is shown exactly as stated above. This way I have a hard time to find all the URLs I should allow. Is there any way to find to display the exact URL that was blocked?

**Comment by dbuechel** [comment, marked as answer]
You'll need to enable the displaying and logging of URLs, see configuration option "Show URLs" under tab "Browser" https://safeexambrowser.org/windows/win_usermanual_en.html#BrowserPane. While configuring this use case, I'd furthermore recommend enabling access to the application log so that you can see in real time what's happening behind the scenes (i.e. which URLs are blocked and which not): https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane

**Reply by danrot** [reply]
Thank you very much, that worked!
